src/corecel/math/SoftEqual.hh

```diff
@@ -19,10 +19,21 @@ namespace celeritas
 {
 //---------------------------------------------------------------------------//
 /*!
- * Functor for floating point equality.
+ * Functor for noninfinite floating point equality.
+ *
+ * This function-like class considers an \em absolute tolerance for values near
+ * zero, and a \em relative tolerance for valuse far from zero. It correctly
```

Comment:
Found this typo on a merged commit.

Reply:
Thanks! I'll fix in an upcoming PR.

Reply:
https://github.com/celeritas-project/celeritas/pull/933/commits/14a7b9fb4da9c652fd53bb1daae91e88066be434